cmd/juju/application/deployer/bundle.go

```diff
@@ -173,6 +178,28 @@ Please repeat the deploy command with the --trust argument if you consent to tru
 	return nil
 }
 
+// checkExplicitBase returns an error if the image-id constraint is used and
+// there is no base (series) explicitly defined by the user.
+func (d *deployBundle) checkExplicitBase(bundleData *charm.BundleData) error {
+	for _, applicationSpec := range bundleData.Applications {
+		charmURL, err := resolveCharmURL(applicationSpec.Charm, d.defaultCharmSchema)
```

Comment:
sure, I had it because I copied what's done in https://github.com/juju/juju/blob/develop/cmd/juju/application/deployer/bundle.go#L118, it's removed now